engines/account/app/services/account/signup_service.rb

```diff
@@ -23,7 +20,7 @@ def create!(email)
     # @return [Account::User]
     def complete!(signup, options)
       ::ActiveRecord::Base.transaction do
-        user = ::Account::User.create(
+        user = ::Account::User.create!(
           email: signup.email,
           username: options[:username],
           password: options[:password],
```

Comment:
@andresgutgon could you please paste here a diff with what we found out about the `SignupService`?